docs/src/appendix/snfoundry-toml.md

```diff
@@ -0,0 +1,103 @@
+# The Manifest Format
+
+The `snfoundry.toml` file, present in each package, is called its manifest. 
+This file allows you to specify sncast settings. It is written in the [TOML](https://toml.io/) format. 
+It contains metadata needed to configure sncast settings and should be placed in the root of your project. 
+If `snfoundry.toml` is not found in the root directory, sncast will look in all parent directories. 
+If it is not found, default values will be used.
+
+Schema of the `snfoundry.toml` file:
```

Comment:
This isn't really a schema. Let's name it something like 

``## `snfoundry.toml` contents``

and update headings bellow accordingly